src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/decisionmaker/deciders/configs/DeciderConfig.java

```diff
@@ -26,7 +24,8 @@
  *     // Decreasing order of priority for the type of workload we can expect on the cluster.
  *     // Priority order in the list goes from most expected to the lease expected workload type.
```

Comment:
We might need to update this comment right?

Reply:
done